Sources/TSCUtility/CMakeLists.txt

```diff
@@ -47,6 +47,13 @@ target_link_libraries(TSCUtility PUBLIC
   TSCBasic)
 target_link_libraries(TSCUtility PRIVATE
   TSCclibc)
+
+if(CMAKE_SYSTEM_NAME STREQUAL Linux OR CMAKE_SYSTEM_NAME STREQUAL Darwin)
```

Comment:
Are the lack of these libraries caused failures on Darwin?  I think the pthread library is brought in through libSystem.

Reply:
No, `pthread` is force linked by the `LC_DYLIB_REEXPORT` of `libSystem_pthread.dylib` from `libSystem`.  This is really a non-Darwin path that it impacts.  I think that something like this is better suited though:

```cmake
find_package(Threads)
target_link_librareis(TSCUtility PRIVATE
  ${CMAKE_DL_LIBS}
  Threads::Threads)
```

Note the fact that this doesn't care about the platform or configuration because that concern is entirely alleviated by offloading that to CMake.

Reply:
Sounds good.  My question was specifically reacting to the `CMAKE_SYSTEM_NAME STREQUAL Darwin` part of the condition.